tests/BootstrapTest.php

```diff
@@ -48,8 +50,63 @@ public function testWorkingFromChildWorkingDirectory()
         $this->assertStringEndsNotWith("/tests", getcwd());
         $this->assertStringStartsNotWith(".././", getenv("LOTGD_CONFIG"));
     }
+    
+    public function testBootstrapLoadsPackageModels()
+    {
+        $installationManager = $this->getMockBuilder(InstallationManager::class)
+            ->disableOriginalConstructor()
+            ->setMethods(["getInstallPath"])
+            ->getMock();
+        $installationManager->method("getInstallPath")->willReturn(__DIR__ . "/FakeModule");
+        
+        $composer = $this->getMockBuilder(\Composer\Composer::class)
+            ->disableOriginalConstructor()
+            ->setMethods(["getInstallationManager", "translateNamespaceToPath"])
+            ->getMock();
+        $composer->method("getInstallationManager")->willReturn($installationManager);
+        $composer
+            ->expects($this->exactly(1))
+            ->method("translateNamespaceToPath")
+            ->with("LotGD\\Core\\Tests\\FakeModule\\Models")
+            ->willReturn(__DIR__ . "/FakeModule/Models");
+        
+        $fakeModulePackage = $this->getMockBuilder(AliasPackage::class)
```

Comment:
Why AliasPackage instead of PackageInterface? I actually dont know what an AliasPackage is.